common/include/pcl/pcl_macros.h

```diff
@@ -206,6 +206,7 @@
   #define M_2_SQRTPI 1.12837916709551257390   // 2/sqrt(pi)
   #define M_SQRT2    1.41421356237309504880   // sqrt(2)
   #define M_SQRT1_2  0.707106781186547524401  // 1/sqrt(2)
+  #define M_PHI      1.61803398874989484820   // golden_ratio
```

Comment:
It might be best to define phi locally in `golden_section_search`, e.g. like this: `constexpr float phi = 1.61803398874989484820f;`